pkg/framework/aries/default_test.go

```diff
@@ -20,6 +20,10 @@ func TestDefaultFramework(t *testing.T) {
 		defer cleanup()
 		dbPath = path
 
+		defMKPath = path + "/mk"
```

Comment:
I see that this is only tested during non-wasm builds.

Should be deferred into another PR.

Reply:
removed